projects/storefrontlib/src/cms-structure/outlet/outlet.providers.ts

```diff
@@ -0,0 +1,41 @@
+import { InjectionToken, StaticProvider, Type } from '@angular/core';
+import { OutletPosition } from './outlet.model';
+
+/**
+ * @private We plan to drive the outlets by standard configuration
+ */
+export const PROVIDE_OUTLET_OPTIONS = new InjectionToken<ProvideOutletOptions>(
```

Comment:
are we still planning? I believe we do it now

Reply:
By standard configuration I mean:
```typescript
ConfigModule.withConfig({
   outlets: {
      'cx-cart-item.configurator': {
          position: OutletPosition.REPLACE,
          component: CartItemOutletConfiguratorComponent
      }
   }
})
```

The token `PROVIDE_OUTLET_OPTIONS ` is private because it's only an implementation detail, not meant to be used by customers. They should use `provideOutlet`. 
In future we can change the implementation of the `provideOutlet` to do `provideConfig({ outlets: { .... } })` inside.

Reply:
aha, got it and agreed